user/tlu/exe/src/euCliTluReader.cxx

```diff
@@ -118,27 +118,27 @@ int main(int /*argc*/, const char **argv) {
 	  auto timeStampEnd = subev->GetTimestampEnd();
           auto particles = subev->GetTag("PARTICLES", "NAN");
 	  auto triggersFired = subev->GetTag("TRIGGER" , "NAN");
-          auto scaler0 = ev->GetTag("SCALER0", "NAN"); 
-          auto scaler1 = ev->GetTag("SCALER1", "NAN"); 
-          auto scaler2 = ev->GetTag("SCALER2", "NAN");
-          auto scaler3 = ev->GetTag("SCALER3", "NAN");
-          auto scaler4 = ev->GetTag("SCALER4", "NAN");
-          auto scaler5 = ev->GetTag("SCALER5", "NAN");
-          auto finets0 = ev->GetTag("FINE_TS0", "NAN");
-          auto finets1 = ev->GetTag("FINE_TS1", "NAN");
-          auto finets2 = ev->GetTag("FINE_TS2", "NAN");
-          auto finets3 = ev->GetTag("FINE_TS3", "NAN");
-          auto finets4 = ev->GetTag("FINE_TS4", "NAN");
-          auto finets5 = ev->GetTag("FINE_TS5", "NAN");
-          std::cout << runNumber << "," << 
-      		   eventNumber << "," << 
-		   triggerNumber << "," << 
-		   timeStampBegin << "," << 
-		   timeStampEnd << "," << 
+          auto scaler0 = subev->GetTag("SCALER0", "NAN"); 
```

Comment:
can't we just

```
d2->SetTag("FINE_TS0", d1->GetTag("FINE_TS0", "NAN"));
```

Or just keep it, might be cleaner. Is there a reason these fine TS are not used to calculate the actual time more finely than the 40MHz?

Reply:
Nah, it's alright.

When you say "EUDAQ2 is agnostics" who isn't then? How do you calculate this downstream?

Reply:
Currently, we're only using the coarse timestamp (as before).

We didn't decide on a more clever way if implementing/configuring this yet.

Reply:
Alright. As soon as we do, this would be the place to do so I guess...

Reply:
I agree, then we'd either need to implement some logic like "average over all which are non-zero" (but they are zero correctly sometimes) or make it configurable.

Let's postpone this discussion until we actually need it.